### ISSUE TITLE
Feature: Temperature sensor

### ISSUE THREAD

**Issue by binary10** [google/science-journal#34]
It would be great if this app could also record temperature (with a legal disclaimer to refrain from using your phone as an oven thermometer).

**Comment by dsaff** [comment]
Agreed it would be fun.  Our initial research indicated that it wouldn't be very useful in the ways that people anticipate: most phones with a temperature sensor are actually measuring chip temperature, and secondarily the temperature of your pocket or hand, with little relationship to ambient temperature.

Do you know of devices where this is less of an issue?
